web/pandas/interactive_terminal/README.md

```diff
@@ -0,0 +1,17 @@
+# The interactive `pandas` terminal
+
+An interactive terminal to easily try `pandas` in the browser, powered by JupyterLite.
+
+## Build
+
+Building the terminal is integrated in the main `pandas_web.py` build script.
+
+In `web/pandas`, run:
+
+```bash
+# install the dependencies
+python -m pip install -r requirements.txt
+
+# build the website
+python pandas_web.py
+```
```

Comment:
Maybe a very short note on what's each config file, so pandas devs don't need to get into the details of JupyterLite for quick changes? So, they know what belongs to each. Or maybe a link to the config reference. Up to you, but I think something like this shoild be useful.